test/CheckoutSdkTest/Disputes/DisputesIntegrationTest.cs

```diff
@@ -12,7 +13,9 @@ private async Task ShouldQueryDisputes()
         {
             var request = new DisputesQueryFilter
             {
-                Limit = 250
+                Limit = 250,
+                To = DateTime.UtcNow,
```

Comment:
Can you please assert that in the `response` object we have the `TO` object?
Just to confirm that this is working as expected, also if this is working, we probably don't need the `from` object.

Reply:
If the object request has not the field From, API return this message:"{"request_id":"0HMEQ8E6D4AK7:00000009","error_type":"request_invalid","error_codes":["paging_from_invalid"]}"

Reply:
That is not really the root cause, the `paging_from_invalid` error means that the date doesn't contains the required format.
Which means that if you receive that is because or either you are sending a value on `from` property or the `to` date format is wrong, 

But `from` is not a mandatory field.